client/layout/poll-invitation/index.js

```diff
@@ -0,0 +1,145 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import snakeCase from 'lodash/snakeCase';
+import includes from 'lodash/includes';
+import memoize from 'lodash/memoize';
+let debug = require( 'debug' )( 'calypso:poll-invitation' );
+
+/**
+ * Internal dependencies
+ */
+let user = require( 'lib/user' )();
+import { ga as googleAnalytics } from 'analytics';
+import Gridicon from 'components/gridicon';
+import { tracks } from 'analytics';
+import config from 'config';
+import preferencesActions from 'lib/preferences/actions';
+
+const _preferencesKey = 'dismissedBrazilianSurvey';
+const _sectionWhiteList = [ 'reader', 'sites' ];
+
+function shouldDisplay() {
+	return 'pt-br' === user.get().localeSlug;
+}
+
+function recordEvent( eventAction ) {
+	googleAnalytics.recordEvent( 'Translator Invitation', eventAction );
+	let tracksEventName = 'calypso_poll_invitation_' + snakeCase( eventAction );
+	debug( 'recording event ' + tracksEventName );
+	tracks.recordEvent( tracksEventName );
+}
+
+let recordEventOnce = memoize( recordEvent );
+
+export default React.createClass( {
+	displayName: 'PollInvitation',
+
+	propTypes: {
+		isVisible: React.PropTypes.bool
+	},
+
+	getInitialState: function() {
+		return {
+			disabled: ! shouldDisplay()
+		};
+	},
+
+	getDefaultProps: function() {
+		return {
+			isVisible: true
+		};
+	},
+
+	render: function() {
+		if ( this.state.disabled ) {
+			debug( 'hiding: has been disabled' );
+			return null;
+		}
+
+		if ( ! this.props.isVisible ) {
+			debug( 'hiding: hidden by parent' );
+			return null;
+		}
+
+		if ( ! config.isEnabled( 'brazil-survey-invitation' ) ) {
+			debug( 'hiding: not enabled in config' );
+			return null;
+		}
+
+		if ( ! this.props.preferences ) {
+			debug( 'hiding: user preferences not loaded' );
+			return null;
+		}
+
+		if ( this.props.preferences[ _preferencesKey ] ) {
+			debug( 'hiding: user has dismissed invitation' );
+			return null;
+		}
+
+		if ( ! includes( _sectionWhiteList, this.props.section.name ) ) {
+			debug( 'hiding: invalid section', this.props.section.name );
+			return null;
+		}
+
+		recordEventOnce( 'Displayed' );
+
+		let subComponents = {
```

Comment:
`const` here instead of `let`. Do we really need to declare an object here, versus simply inlining the text within the rendered return value?